newsletter-april-2021.rst

```diff
@@ -0,0 +1,66 @@
+.. post:: April 5, 2021
+   :tags: newsletter
+   :author: Juan Luis
+   :location: Madrid
+
+.. meta::
+   :description lang=en:
+      xxx
+
+Read the Docs newsletter - April 2021
+=====================================
+
+xxx
+
+Company highlights
+------------------
+
+-  We have a new colleague! Juan Luis will be working with us as
+   Developer Advocate to …
+-  xxx we saw an increment of X % page views
+-  We have `a new status page <http://status.readthedocs.com/>`__ so you
+   can have a glance at how our service is working.
+
+New features
+------------
+
+-  There is a `new support form <https://readthedocs.org/support/>`__ in
+   case you need to send us a request related to your project or
+   account, like asking for more resources, change the project slug, and
+   so forth.
+-  You can now build your project using Python 3.9 by setting your build
+   image to ``testing``. Please `open an
+   issue <https://github.com/readthedocs/readthedocs.org/issues/new>`__
+   if you find any problems with it.
+-  We added the possibility to use the latest version of MkDocs. `Let us
```

Comment:
I believe this is on by default for new projects, but we can double check that in the admin.

Reply:
yeah, this is on by default for new projects. Note that users were able to use the latest version of mkdocs, it just wasn't the default version (they needed to use a requirements file with `mkdocs==1.1.x`)